docs/during/timetool.md

```diff
@@ -18,7 +19,15 @@ Before the time tool can be employed it must be calibrated. During the process o
 
 (TBA Thursday 2023/03/02)
 
-## Analysis
+![Timetool - DAQ control interface](./material/DAQ_control_TT.png)
```

Comment:
Maybe we can make a more structured figure environment with captions?

I think the answer here: https://stackoverflow.com/questions/19331362/using-an-image-caption-in-markdown-jekyll that uses tables is not a bad option. Something like this?

```
| ![Timetool - DAQ control interface](./material/DAQ_control_TT.png) | 
|:--:| 
| *__DAQ control interface to the Timetool.__ In the `DAQ Control` panel, (1) click `Edit`, (2) click `OPAL1K_1` or some equivalent device name, (3) click on `TimeToolConfig`. This will open the window on the left.* |
```

Doesn't necessarily need to be centered.

Reply:
Good idea, done!